Formula/mk.rb

```diff
@@ -0,0 +1,90 @@
+class Mk < Formula
+  include Language::Python::Virtualenv
+
+  desc "Developer friendly build/test CLI launcher"
+  homepage "https://github.com/pycontribs/mk"
+  url "https://files.pythonhosted.org/packages/fe/cd/fe07815319409d39c7b7506e865708caa289a82126f683b63c2e51ff3428/mk-1.0.2.tar.gz"
+  sha256 "6637f035ddf92a43d2daa5264ce3bfbb5c7e937a3967a1618e48f7247a6914d0"
+  license "MIT"
+  revision 1
+
+  depends_on "python@3.9" => :build
+
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.9.0.tar.gz"
+    sha256 "e24052411fc4fbd1f672635537c3fc2330d9481b18c0317695b46259512c91d5"
+  end
+
+  resource "blessings" do
+    url "https://files.pythonhosted.org/packages/5c/f8/9f5e69a63a9243448350b44c87fae74588aa634979e6c0c501f26a4f6df7/blessings-1.7.tar.gz"
+    sha256 "98e5854d805f50a5b58ac2333411b0482516a8210f23f43308baeb58d77c157d"
+  end
+  resource "click-help-colors" do
```

Comment:
Let's follow other formulae here and put one line of whitespace in between resource blocks.

When you make this change, please squash it with your first commit and use the commit message
```
mk 1.0.2 (new formula)
```

For future reference, please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings. Refer to the [documentation](https://docs.brew.sh/Formula-Cookbook#commit) for more details.